_source/_posts/2022-09-08-react-fastify-postgres.md

```diff
@@ -0,0 +1,897 @@
+---
+layout: blog_post
+title: ""
+author: gabi-dombrowski
+by: contractor
+communities: [javascript]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+---
+
+In this tutorial, we'll set up an app to help us keep track of what NASA facilities we've visited and which ones we still want to visit.
+
+Our app will be a monorepo with Okta authentication using React for the frontend and Fastify for the backend. [Fastify](https://www.fastify.io/) is a highly performant web framework with low overhead that we'll connect to a Postgresql database. We'll also use [Lerna](https://lerna.js.org/) to manage the frontend and backend apps in a monorepo.
+
+**Prerequisites**
+
+*Create React App* currently requires Node >= 14.0.0 and npm >= 5.6. The latest required versions can be found at [https://reactjs.org/docs/create-a-new-react-app.html](https://reactjs.org/docs/create-a-new-react-app.html).
+
+{% include toc.md %}
+
+## Add authentication using OAuth2 and OpenID Connect (OIDC)
+
+For this demo app, we'll be using [Okta's SPA redirect model](https://developer.okta.com/docs/guides/sign-into-spa-redirect/react/main/) to authenticate and fetch user info.
```

Comment:
I think we should add some phrasing about creating the project directory, such as `okta-react-fastify`